haskell/cabal.bzl

```
@@ -345,11 +346,11 @@ def _prepare_cabal_inputs(
         runghc_args = runghc_args,
         extra_args = extra_args,
         path_args = path_args,
-        toolchain_info = _cabal_toolchain_info(hs, cc, workspace_name),
+        toolchain_info = _cabal_toolchain_info(hs, cc, workspace_name, runghc),
     )
 
     inputs = depset(
-        [setup, hs.tools.ghc, hs.tools.ghc_pkg, hs.tools.runghc],
+        [setup, hs.tools.ghc, hs.tools.ghc_pkg, runghc],
```

Does `runghc` still need to be added to `inputs`, if it's added to `tools` and `input_manifests` later?

---

I've tried and it seems to work.
```suggestion
        [setup, hs.tools.ghc, hs.tools.ghc_pkg],
```